boards/FSM/Inc/App/App_AcceleratorPedals.h

```diff
@@ -1,6 +1,8 @@
 #pragma once
 
 #include <stdbool.h>
+#include "App_FsmWorld.h"
+#include "App_Brake.h"
 
 #define PAPPS_ENCODER_FULLY_PRESSED_VALUE (937U)
```

Comment:
Maybe you can leave a comment on how you picked these

Reply:
No need for brakes header file any longer 🎉🎉 f249f18

Reply:
also it was included in FsmWorld anyways idk why I included it again